astropy/convolution/convolve.py

```diff
@@ -784,3 +786,37 @@ def interpolate_replace_nans(array, kernel, convolve=convolve, **kwargs):
     newarray[isnan] = convolved[isnan]
 
     return newarray
+
+
+def convolve_models(model, kernel, mode='convolve_fft', **kwargs):
+    """
+    Convolve two models using `~astropy.convolution.convolve_fft`.
+
+    Parameters
+    ----------
+    model : `~astropy.modeling.core.Model`
+        Functional model
+    kernel : `~astropy.modeling.core.Model`
+        Convolution kernel
+    mode : str
+        Keyword representing which function to use for convolution.
+            * 'convolve_fft' : use `~astropy.convolution.convolve_fft` function.
+            * 'convolve' : use `~astropy.convolution.convolve`.
+    kwargs : dict
+        Keyword arguments to me passed either to `~astropy.convolution.convolve`
+        or `~astropy.convolution.convolve_fft` depending on ``mode``.
+
+    Returns
+    -------
+    default : CompoundModel
+        Convolved model
+    """
+
+    if mode == 'convolve_fft':
+        BINARY_OPERATORS['convolve_fft'] = _make_arithmetic_operator(partial(convolve_fft, **kwargs))
+    elif mode == 'convolve':
+        BINARY_OPERATORS['convolve'] = _make_arithmetic_operator(partial(convolve, **kwargs))
+    else:
+        raise ValueError('Mode ' + mode + ' is not supported.')
```

Comment:
maybe use string formatting here, rather than addition, in case a use tries to pass `None` or some other non-string object?